.github/workflows/test.yaml

```diff
@@ -2,8 +2,12 @@ name: CI
 
 on:
   push:
+    paths-ignore:
+      - 'README.md'
```

Comment:
May I know why you need to ignore README?

Reply:
Since changes to the README doesn't affect the outcome of the test.

Reply:
But can a change to README affect the outcome of the tests? I don't think tests use README as their input in any way.

Reply:
Okay.